include/natalie/string_object.hpp

```diff
@@ -189,6 +189,6 @@ class StringObject : public Object {
     void raise_encoding_invalid_byte_sequence_error(Env *, size_t) const;
 
     String m_string {};
-    Encoding m_encoding { Encoding::UTF_8 };
+    Encoding m_encoding { Encoding::ASCII_8BIT };
```

Comment:
Can you explain why this change was necessary?

Reply:
`String::allocate` (and `String::new` without an `encoding:` argument) default to ASCII-8BIT; I've found a better way to do this